shell/models/__tests__/management.cattle.io.node.ts

```diff
@@ -0,0 +1,82 @@
+import MgmtNode from '@shell/models/management.cattle.io.node';
+
+describe('class MgmtNode', () => {
+  const t = jest.fn(() => 'baz');
+  const ctx = { rootGetters: { 'i18n/t': t } };
+
+  const resetMocks = () => {
+    // Clear all mock function calls:
+    jest.clearAllMocks();
+  };
+
+  it('should not return addresses if they are not present in the resource status, the internalNodeStatus, or the rkeNode key in status', () => {
+    const mgmtNode = new MgmtNode({ status: {} }, ctx);
```

Comment:
This should be `undefined` based on the code.

Reply:
actually, it should return an empty array which is what it's doing and what I'm testing for: 
<img width="556" alt="image" src="https://github.com/rancher/dashboard/assets/37550899/f9b8e2de-9891-412d-bcdb-d81c83482e5b">